README.md

```diff
@@ -1,26 +1,177 @@
+  first_release_date: 2000-01-01
+  first_release_version: 0.1 # Reference: https://example.com/releases
+  latest_release_date: 2024-12-28
+  latest_release_version: 0.37.2 # Reference: https://example.com/releases
+```
+
+If the repository is available but not on GitHub, you can add the `repo_url` key.
+
+```yaml
+- name: Project Name
+  url: https://example.com
+  repo_url: https://gitlab.com/example
+  first_release_date: 2000-01-01
+  first_release_version: 0.1 # Reference: https://example.com/releases
+  latest_release_date: 2024-12-28
+  latest_release_version: 0.37.2 # Reference: https://example.com/releases
+```
+
+If the release count is available you can add the `release_count` key.
+
+```yaml
+- name: Project Name
+  url: https://example.com
+  repo_url: https://gitlab.com/example
+  first_release_date: 2000-01-01
+  first_release_version: 0.1 # Reference: https://example.com/releases
+  latest_release_date: 2024-12-28
+  latest_release_version: 0.37.2 # Reference: https://example.com/releases
+  release_count: 100 # Reference: https://example.com/releases
+```
+
+### Emeritus Complex GitHub Projects or Emeritus Non-GitHub Projects
+
+If a complex GitHub project or non-GitHub project is no longer ZeroVer and needs to be added to the emeritus list, you will need to change several keys.
+
+```yaml
+- name: Project Name
+  url: https://example.com
+  emeritus: true
+  first_release_date: 2000-01-01
+  first_release_version: 0.1 # Reference: https://example.com/releases
+  first_nonzv_release_date: 2024-12-30
+  first_nonzv_release_version: 1.0.0
+  last_zv_release_version: 0.37.2 # Reference: https://example.com/releases
+  release_count_zv: 100 # Reference: https://example.com/releases
+```
+
 ## Building
 
-The site is statically generated using
-[Chert](https://github.com/mahmoud/chert).
+1. Create and enter a virtual environment if you haven't already
+2. `pip install -r requirements.txt`
+
+### Updating projects.json
+
+1. Obtain a [personal access token from GitHub](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token). This token will not need any special permissions, the default options should be fine.
+2. Create a file named `gh_token` and paste in your personal access token.
+3. Run `python tools/gen_projects_json.py --user YOUR_USERNAME --token gh_token`
+
+### Serving the site
+
+Simply run `chert serve`.
 
-Content is all under `entries/`.
+## CI/CD
 
-Tables of current and past ZeroVer users are generated from
-`projects.yaml` into `projects.json` by `tools/gen_projects_json.py`
-using the GitHub API. The project list has gotten long enough that
-you'll need an API key.
+0ver uses [GitHub Actions](https://github.com/features/actions) to validate `projects.yaml`, update `project.json`, and publish new versions of the site. To configure CI/CD in a fork, you only need to set the environment variables.
```

Comment:
Actually, I think it would be better to use the automatically-provisioned [GITHUB_TOKEN](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication) here, if we can. That'll work for everyone.

Reply:
Oh yeah, I saw that and forgot to circle back to see if it would work. I'll look into that now.

Reply:
I'm so confused, I am following [the docs](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#example-1-passing-the-github_token-as-an-input) precisely and it's [not working](https://github.com/MattTheCuber/zerover/actions/runs/12563301344/job/35024961566)...

Reply:
Aha, I forgot the GH_USER.